geotrek/api/v2/serializers.py

```diff
@@ -1406,15 +1406,27 @@ class Meta:
             model = signage_models.Blade
             fields = ('id', 'number', 'color', 'direction', 'lines')
 
+    class SignageConditionSerializer(DynamicFieldsMixin, serializers.ModelSerializer):
+
+        class Meta:
+            model = signage_models.SignageCondition
+            fields = ('id', 'label', 'structure')
+
     class SignageSerializer(DynamicFieldsMixin, serializers.ModelSerializer):
         geometry = geo_serializers.GeometryField(read_only=True, source="geom3d_transformed", precision=7)
         structure = serializers.CharField(source='structure.name')
         attachments = AttachmentSerializer(many=True)
         blades = BladeSerializer(source='blades_set', many=True)
+        condition = serializers.SerializerMethodField(
+            help_text=_("This field is deprecated and will be removed in next releases. Please start using 'conditions'")
+        )
+
+        def get_condition(self, obj):
+            return str(obj.conditions.first())
```

Comment:
for retrocompat, condition should be an integer

https://demo-admin.geotrek.fr/api/v2/signage/?fields=id,condition